SilKit/source/core/internal/traits/SilKitMsgTraits.hpp

```diff
@@ -114,7 +116,8 @@ struct SilKitMsgTraits
         { \
             return true; \
         } \
-    };
+    }
```

Comment:
nice catch. hope this is the only place where we added semicolons to macros...

Reply:
I skimmed through the other trait headers and didn't see one.